brave/renderer/brave_content_renderer_client.cc

```diff
@@ -164,6 +183,17 @@ void BraveContentRendererClient::RenderViewCreated(
   new ChromeRenderViewObserver(render_view, web_cache_impl_.get());
 
   new password_manager::CredentialManagerClient(render_view);
+
+#if BUILDFLAG(ENABLE_SPELLCHECK)
+  // This is a workaround keeping the behavior that, the Blink side spellcheck
+  // enabled state is initialized on RenderView creation.
+  // TODO(xiaochengh): Design better way to sync between Chrome-side and
+  // Blink-side spellcheck enabled states.  See crbug.com/710097.
+  if (SpellCheckProvider* provider =
+          SpellCheckProvider::Get(render_view->GetMainRenderFrame()))
+    provider->EnableSpellcheck(spellcheck_->IsSpellcheckEnabled());
+  new SpellCheckPanel(render_view);
```

Comment:
yes, we should. because I was working based on cr59 and this is added in cr60.
When I pushed the commit, I just rebase the muon for our cr60 commits